src/nfts/ducks/index.ts

```diff
@@ -10,14 +10,23 @@ import {
 } from 'nfts/ducks/constants';
 import * as React from 'react';
 import { AssetDetail } from 'assets/types';
+import { NftConfig } from '../../constants';
 
 export class Duck extends BaseNft<DuckInfo> {
   protected genoType: string;
   protected generationColor: string;
   protected druck: string | null;
 
-  constructor(asset: AssetDetail, info: DuckInfo) {
-    super(asset, info);
+  constructor({
+    asset,
+    info,
+    config,
+  }: {
+    asset: AssetDetail;
+    info?: DuckInfo | null;
+    config?: NftConfig | null;
```

Comment:
Doesn't seem to be ok that we can have no config for nfts. We have a default value anyway, right?

Reply:
Of course, config is always set with the default value in the redux state.
Just a mistake.